spec/ttml1.xml

```diff
@@ -7070,20 +7070,19 @@ set of <emph>E</emph>, <emph>SSS(E)</emph>;</p></item>
 <item>
 <p><phrase role="strong">[implicit inheritance]</phrase> if the
 element type of <emph>E</emph> is not the styling element type
-<el>style</el>, then for each inheritable style property
-<emph>P<sub>INH</sub></emph> in the set of style properties defined
+<el>style</el>, then for each style property
+<emph>P</emph> in the set of style properties defined
 above in <specref ref="styling-attribute-vocabulary"/>, perform the
 following ordered sub-steps:</p>
 <olist>
-<item><p>if <emph>P<sub>INH</sub></emph> is present in the specified
+<item><p>if <emph>P</emph> is present in the specified
 style set of <emph>E</emph>, <emph>SSS(E)</emph>, then continue to the
-next inheritable style property;</p></item>
-<item><p>if the element type of <emph>E</emph> is the layout element
-type <el>region</el>, then set <emph>P&prime;</emph> to the initial value of
+next style property;</p></item>
+<item><p>if <emph>P</emph> is inheritable and <emph>E</emph> is a <el>region</el> element, or if <emph>P</emph> is not inheritable and applies to <emph>E</emph>, then set <emph>P&prime;</emph> to the initial value of
 property <emph>P</emph>, where the initial value of a property is
 determined according to the specific property definition found above
 in <specref ref="styling-attribute-vocabulary"/>;</p></item>
```

Comment:
Even if not using my suggestion at https://github.com/w3c/ttml1/pull/298/files#r157463436 right now, I think it would still be useful to note that if an initial value is absent from the property definition, then the specified value falls into the undefined class.

Reply:
Do you mean replace _then set P′ to the initial value of property P,_ with _then set P′ to the initial value of property P if one exists,_ ?

Reply:
That would work but it would be a bigger change, since it would also mean that the step "if the value of P′ is not undefined, then merge P′ into the specified style set of E, SSS(E)." would need to be changed as well.

The smaller (and therefore usually preferable) change is simply to add something like the following to the end of the line this comment thread begins on:

_(for properties that do not define an initial value, the value of P′ is undefined)_

Reply:
@nigelmegitt if no initial value of property P exist than P' is not set, therefore P' remains undefined.... or is there something I missed?

Reply:
@nigelmegitt _then set P′ to the initial value of property P if one exists_ seems to cover the case where no initial value is defined, since the clause would not be applied if no initial value exists, which would leave P' undefined... or am I missing a corner case... or is it a personal style issue?

Reply:
@palemieux Maybe I've misunderstood, but I thought that if you do that ["if one exists"] then SSS(E) would not contain P', whereas if you adopt my approach SSS(E) would contain P', with P' set to an "undefined" class. So it seemed like a subtle but substantive difference.

Reply:
@nigelmegitt There is no "undefined" class in SSS(E). AFAIK "undefined" refers to the variable P' never having been set.

Reply:
@palemieux OK, I see what you mean. If P' is undefined then it never gets added to SSS(E). So we should say:

_set P′ to the initial value of property P if such an initial property is defined_

I'm wary of saying "if one is defined" in case of ambiguity about what the "one" refers to.

Reply:
@nigelmegitt I have updated the PR per https://github.com/w3c/ttml1/pull/298#discussion_r157547379